### DISCUSSION TITLE
PatchTool 6.18 7/6

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#188] [Announcements]
修正翻譯錯誤，並跟進到6.18版本。部分漢化（如技能說明）尚未更新。

請注意不要在已漢化的情況下更新遊戲。

檔案：
- FFXIVChnTextPatch-0706.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：
- rawexd-zhcn-0706.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [第三方4K字體包](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.53/font-4k.rar)：部分解析度設定下會出現字體錯位等情形，請自行照需求使用。解壓後改名為font放到resource資料夾下（取代原資料夾）。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- backup-0706.rar：乾淨的0a0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.65'>PatchTool 6.18 7/6</a>.</em>

**Comment by okami2223** [comment]
請問backup-0706.rar 檔案在哪呢?

**Reply by GpointChen** [reply]
感謝回報，忘記上傳了，已更新。

**Reply by VUV-DI** [reply]
不好意思 請教一下 這次漢化內容是否只包含到萬魔殿而已

6.1 大象蠻族 媽媽老主顧沒漢化到 不確定是不是更新失敗了

**Reply by GpointChen** [reply]
是的

**Reply by VUV-DI** [reply]
> 是的

謝謝你的回應 還擔心是我安裝失敗才缺

**Reply by notnivecs** [reply]
不好意思打扰一下.... 请问您这边有推荐的在steamOs上安装汉化的方式吗？ 目前试了一下用steam自带的proton 但是似乎会出现安装界面黑掉而导致无法选取路径的问题... 谢谢...

**Comment by GpointChen** [comment]
8/9 預計更新中國服文本

**Reply by tonyjediex** [reply]
謝謝大佬

**Reply by ClareTW** [reply]
不好意思，請問中文化不是下載
000000，0a0000
那六個檔案覆蓋過去就好了嗎？
有大佬能提供那六個檔案嗎？(不好意思，macos M1版本，試了好久，一直沒辦法自己用大佬們教的方法弄出檔案)

**Comment by KppKerry** [comment]
感謝大佬
